en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -430,99 +424,97 @@ Similar to the example that was walked through above, [`PH_nandeck_Your_Timeline
 
-So before we can upload our _Timeline_ deck into TTS, we will need to regenerate our cards into this required format. To do this, it is necessary to split our original spreadsheet into two spreadsheets. The first spreadsheet will generate the deck's 'face' and as such, will have the necessary information to generate facts, images and years. This spreadsheet will no longer need duplicate entries that missing entries in the 'years' column. The second spreadsheet will be identical to the first with columns for 'Fact' and 'Image', but will not include the 'Year.' 
+Before we can upload our _Timeline_ deck into TTS, we will need to regenerate our cards into this required format. To do this, we have to split our original spreadsheet into two. The first spreadsheet will generate the deck's faces and, as such, will have the necessary information to generate facts, images and years. This spreadsheet will no longer need the duplicate entries without the 'years'. The second spreadsheet will be identical to the first, except that it won't include the 'year' column. 
 
-Then it is necessary to change our script to reflect that the spreadsheets are going to be used to generate one of two files needed by TTS:
+We will have to change our script to reflect that the spreadsheets are going to be used to generate one of two files needed by TTS:
 
-First, we are going to change the link variable so that it goes to the spreadsheet that will generate the decks' faces.
+First, let's change the `LINK` variable to point to the spreadsheet that will generate the decks' faces.
 
 ```
 LINK = "Windsor-Timeline_for_Tabletop_Face.xlsx"
 ```
 
-As we no longer have to generate one document that meant to be printed and then folded, we can remove these lines from our code:
+Because we are no longer generating one document to be printed and folded, we can remove the following lines from our code:
 
 ```
 DUPLEX = 1-59,60-118
 PRINT = DUPLEX
 ```
 
-TTS requires image files for each side of a deck to be in a standard format of page of cards that run ten in number across, and seven in number down. This standardization is necessary so the program knows how to calculate where the cards should be parsed into separate images. (If your deck has more than 70 cards, it will be necessary to parse the deck into as many as necessary under the 70 card limit). 
+TTS requires the image files for each side of the deck to be provided in a certain standard format: one page containing ten cards across, and seven down. This standardization is necessary for the program to calculate where the cards should be parsed into separate images. (If your deck has more than 70 cards, it will be necessary to parse the deck into as many as necessary under the 70 card limit). 
```

Comment:
I was also a little unsure about the meaning of the section in brackets. Could you provide a slightly rephrased description here as well?

Reply:
Is this better?  "If your deck has more than 70 cards, then you are going to have to run through this exercise several times with generating sets of cards of 70 cards or less. For example, if you would like to generate the card faces of 200 cards, you would need to generate and process three spreadsheets: two spreadsheets of 70 rows each and one spreadsheet of 60 rows."